docs/en/guides/pipeline-integrations/development/sgtk-developer-app.md

```diff
@@ -0,0 +1,257 @@
+---
+layout: default
+title: Developing apps
+pagename: sgtk-developer-app
+lang: en
+---
+
+# Developing your own app
+
+## Introduction
+
+This guide outlines what a Toolkit app is, and covers how to create your own, and also explains some of the basics around app development.
+Shotgun Pipeline Toolkit is not only a collection of apps and engines maintained by Shotgun Software, it is also a development platform that makes it easy to quickly build custom pipeline tools!
+
+- [What is a Toolkit app?](#what-is-a-toolkit-app)
+- [Creating your own app](#creating-your-own-app)
+
+Steps:
+1. [Creating a development sandbox](#part-1---creating-a-development-sandbox)
+2. [Forking or downloading the starter app repository](#part-2---forking-or-downloading-the-starter-app-repository)
+3. [Adding the app to your config](#part-3---adding-the-app-to-your-config)
+4. [Developing the app](#part-4---developing-the-app)
+    - [Anatomy of the Template Starter App](#anatomy-of-the-template-starter-app)
+    - [Configuration settings](#configuration-settings)
+    - [Frameworks](#frameworks)
+    - [Reloading your changes](#reloading-your-changes)
+5. [Testing](#part-5---testing)
+6. [Preparing your first release](#part-6---preparing-your-first-release)
+
+Additional info:
+- [Modifying an existing app](#modifying-an-existing-app)
+    - [Pull requests](#pull-requests)
+
+## What is a Toolkit app?
+
+A Toolkit app can be defined as follows:
+
+- A Tool that is usually run by an end-user, using the Shotgun integrations.
+- They usually have a graphical user interface with which the user can guide the app's operations, but it doesn't have to, it can also be a command registered with the integrations, triggered by the user from the Shotgun menu in the software.
```

Comment:
- I'd say "Apps usually have", since we're referring to them in the singular up to this point
 - and since we are going plural, it should be "but *they* don't have to"
 - I'd suggest "host software" instead of software, again to make it match the other docs
 - and this should be two sentences. 

So all in all:

"Apps usually have a graphical user interface with which the user can guide the app's operations, but they don't have to. An app can also be a command registered with the integrations, triggered by the user from the Shotgun menu in the host software."